### DISCUSSION TITLE
2007/01/26/%E5%B8%A7

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#142] [Announcements]
# 2007/01/26/%E5%B8%A7

不知道是我眼花还是微软的 bug。今天用微软拼音2003试图输入“帧”字，可是怎么找都没找到。最后换了输入法才输入进去。

https://blog.qizhen.xyz/2007/01/26/%E5%B8%A7

**Comment by ruanqizhen** [comment]
之前博客下的留言：


奕瑋
2007年02月15日 09:29 编辑
是的，绝大多数的时候被我们念错了，读成“桢”的音了。

回复
双
2007年02月5日 17:20 编辑
用微软2003是zheng，呵呵

回复
